src/query/plan/preprocess.hpp

```diff
@@ -213,7 +212,7 @@ class PatternFilterVisitor : public ExpressionVisitor<void> {
 /// Stores the symbols and expression used to filter a property.
 class PropertyFilter {
  public:
-  using Bound = ScanAllByLabelPropertyRange::Bound;
+  using Bound = utils::Bound<Expression *>;
```

Comment:
Could you explain me why is this needed?

Reply:
Operator.hpp must include this header because it contains the definition of `Filter::all_filters_`. To prevent a circular dependency, I redefine `Bound` here without relying on operator.hpp’s `ScanAllByLabelPropertyRange`. `Bound` itself is unchanged (compare its definition in `ScanAllByLabelPropertyRange`).